docs/content/examples/use-cases/external-relay-only-mailserver.md

```diff
@@ -0,0 +1,213 @@
+---
+title: 'Use Cases | Use a public server for relaying between a private DMS instance'
+hide:
+  - toc
+---
+
+## Introduction
+
+!!! info "Community contributed guide"
+
+    Adapted into a guide from [this discussion](https://github.com/orgs/docker-mailserver/discussions/3965).
+
+    **Requirements:**
+
+    - A _public server_ with a static IP, like many VPS providers offer. It will only relay mail to DMS, no mail is stored on this system.
+    - A _private server_ (eg: a local system at home) that will run DMS.
+    - Both servers are connected to the same network via a VPN (_optional convenience for trust via the `mynetworks` setting_). We will assume below that the VPN is setup on `192.168.2.0/24`, with the _public server_ using `192.168.2.2` and the _private server_ using `192.168.2.3`.
+
+The goal of this guide is to configure a _public server_ that can receive inbound mail and relay that over to DMS on a _private server_, which can likewise submit mail outbound through a _public server_ or service. The primary motivation is keep your mail storage private, instead of storing unencrypted on a VPS host disk.
+  
+## DNS setup
+
+Follow our [standard guidance][docs::usage-dns-setup] for DNS setup.
+
+!!! example "DNZ Zone file example"
+
+    - A public reachable IP address of `11.22.33.44`
+    - Mail for `@example.com` addreses has an MX record to `mail.example.com` which resolves to that _public server_ IP.
+    - Set your A, MX and PTR records for the _public server_ as if it were running DMS.
+
+    ```txt
+    $ORIGIN example.com
+    @     IN  A      123.123.123.123
+    mail  IN  A      123.123.123.123
+
+    ; mail server for example.com
+    @     IN  MX  10 mail.example.com.
+    ```
+
+    SPF records should also be setup as you normally would for `mail.example.com`.
```

Comment:
```suggestion

Set your A, MX and PTR records for the _public server_ as if it were running DMS.

!!! example "DNZ Zone file example"
    
    For this guide we assume DNS is configured with:

    - A public reachable IP address of `11.22.33.44`
    - Mail for `@example.com` addresses should have an MX record to `mail.example.com` which A record then resolves to the IP of your _public server_.

    ```txt
    $ORIGIN example.com
    @     IN  A      11.22.33.44
    mail  IN  A      11.22.33.44

    ; mail server for example.com
    @     IN  MX  10 mail.example.com.
    ```

    SPF records should also be setup as you normally would for `mail.example.com`.
```